.svglintrc.js

```diff
@@ -145,11 +145,11 @@ module.exports = {
               return;
             }
 
-            const bounds = getBounds(iconPath);
+            const bbox = svgPathBbox(iconPath);
```

Comment:
A matter of preference, but would be clearer destructuring to something like:
```js
const [minX, maxX, minY, maxY] = svgPathBbox(iconPath);
```

Reply:
I agree, it's done :+1: